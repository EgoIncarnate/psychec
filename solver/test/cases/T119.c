void f() {
    const char* s = "a";
    T2 ss = s;
    f1(ss);
    T3 sss = "a";
    f3(sss);
}
