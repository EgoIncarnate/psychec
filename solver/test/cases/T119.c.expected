typedef  char const* T3 ;
typedef  char const* T2 ;
uintptr_t f1 (char const*) ;
uintptr_t f3 (char const*) ;
